modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -119,6 +119,46 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172891. Verify that the start date cannot be equals to the current date"
+	@priority = "5"
+	test StartDateCannotBeEqualsToCurrentDate {
+		task ("Given a new request") {
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When a new activity is created") {
+			PRMMDFRequest.addNewActivity();
+		}
+
+		task ("And a date is set on 'Start Date' field") {
+			PRMMDFRequest.insertDate(
+				controlDate = "-1",
+				element = "dddd");
+		}
+
+		Click(
+			key_fieldName = "activities[0].endDate",
+			locator1 = "PRMMDFRequest#TEXT_FIELD");
+
+		AssertElementPresent(
+			key_className = "form-feedback-item",
+			key_errorMessage = "Start date need to be after today",
+			locator1 = "CreateObject#ERROR_MESSAGE",
+			value1 = "Start date need to be after today");
```

Comment:
Hi @mateussandes23, here I think that you can search for a macro that do an error assertion, or try to create a new one.